java/cuvs-java/src/main/java22/com/nvidia/cuvs/internal/SearchResultsImpl.java

```diff
@@ -0,0 +1,58 @@
+package com.nvidia.cuvs.internal;
+
+import com.nvidia.cuvs.SearchResults;
+
+import java.lang.foreign.MemoryLayout;
+import java.lang.foreign.MemorySegment;
+import java.lang.foreign.SequenceLayout;
+import java.util.LinkedHashMap;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Map;
+
+class SearchResultsImpl implements SearchResults {
```

Comment:
This has been moved, but git does not recognize it as it also changed a bit.
I moved it to this package so I was able to restrict visibility to be package-private.